### ISSUE TITLE
Investigate TechnicLargeAngularMotor (grey) STATS mode

### ISSUE THREAD

**Issue by tthiery** [sharpbrick/powered-up#109]
The `TechnicLargeAngularMotorGrey` has a non-understood STATS mode. Investigate it and extend model.

**Comment by rickjansen-dev** [comment]
I tried the MultiValueMode thing, but it never returns a value. It is possible to call SetupNotificationAsync for the STATS mode, but regardless of the notification setting it never returns a value. I tried to send an PortInformationRequest to get the port value to no avail and if I interpret the lego wireless protocol document correctly, that only works if it's actually an Input mode, which it isn't. So i'm unsure how to actually get a value to be sent by the device.

Also, the `TechnicMediumAngularMotorGrey` has the same `STATS` mode.

**Comment by tthiery** [comment]
As a wider community we desperately need a channel to the LEGO group. I mean, this is not critical ... but in 10 years when LEGO moves on, the community needs to understand every detail of it and not only a part of it.

**Comment by rickjansen-dev** [comment]
I tend to agree with you on this. Although I could somewhat understand that they would not be interested in documenting their own devices like this.

How do you suggest we try and get this information? Contacting Lego through the LWP github repo seems of no use (or at the very least you can wait months if not years for a reply). Maybe simply writing to Lego support and hope to somehow get trough to the people that know?

**Comment by tthiery** [comment]
I hope with the Spike/Mindstorms Hub there will be a spec dump soon. I also hope that one day we have a direct line to their dev team.

Do not worry about this right now. It is named stats ... so not really a big concern about the functionality. For me open issues are not a concern. There are not bugs (which are labeled) but ideas what to do next.
